DennisNjeri/SendIT
# Ft signup page #161582802

**What does this PR do?**
This PR creates a registration form for the user

**Description of Task to be completed?**
Create user account in the application

How should this be manually tested?
Clone this repository and then run file signup.html

Any background context you want to provide?
None

What are the relevant pivotal tracker stories?
A user should be able login to the system. [##161582802
![screenshot from 2018-11-02 12-42-49](https://user-images.githubusercontent.com/42151211/47917455-040de600-de80-11e8-80de-ee108913c502.png)
]
